tests/images/scratch/image.yaml

```diff
@@ -0,0 +1,27 @@
+name: "cekit-scratch"
+version: "1.0.0"
+from: "scratch"
+description: "Minimal scratch example"
+
+labels:
+  - name: "io.cekit.test"
+    value: "This is a CEKit test label"
+
+envs:
+  - name: "CEKIT_TEST"
+    value: "test"
+
+artifacts:
+    # Both files will be added to a /files/ directory in teh container (which will be created)
```

Comment:
teh -> the

Reply:
Thanks! Will fix it in the next round! :)